privacyidea/api/lib/prepolicy.py

```diff
@@ -2104,3 +2104,24 @@ def required_piv_attestation(request, action=None):
         # Add parameter verify_attestation
         request.all_data["verify_attestation"] = REQUIRE_ACTIONS.VERIFY in list(require_att) or \
                                                  REQUIRE_ACTIONS.REQUIRE_AND_VERIFY in list(require_att)
+
+
+def check_excluded_tokeninfo_fields(request=None, action=None):
+    user_object = request.User
+    role = g.logged_in_user.get("role")
+    if role == ROLE.USER:
+        return True
+
+    pols = Match.user(g, scope=SCOPE.USER, action=ACTION.TOKENINFO_API_EXCLUDE, user_object=user_object)\
+        .action_values(unique=True)
```

Comment:
If you set unique=False, you may specify the policy action values as white spaced list (no need to split by comma)

We should make this policy available for admins or users (see below). Check for both:

``` python
    pols = Match.admin_or_user(g, action=ACTION.TOKENINFO_API_EXCLUDE,
                               user_obj=request.User).action_values(unique=False)
```